docs/admin_api/media_admin_api.md

```diff
@@ -159,6 +160,26 @@ Response:
 {}
 ```
 
+## Unprotecting media from being quarantined
+
+This API reverts the protection of a media.
+
+Request:
+
+```
+DELETE /_synapse/admin/v1/media/protect/<media_id>
```

Comment:
does `DELETE` feel scary to anyone else? I feel like I'd be worrying that I'm deleting the media itself.

I'd be inclined towards `POST /_synapse/admin/v1/media/unprotect/<media_id>`, but I don't have a very strong opinion.

Reply:
I do not like so many URI. When you something create with `PUT` or `PULL` you can revert it with `DELETE`. This ist HTTP.
We can add a sentence, that this do not delete a media. It only deletes the protection.
Do you have team call for discussing this? IMO there ist missin a kind of compliance for all oh the admin API.
Probably is need for an extensive API like for user.

Reply:
I agree that `DELETE` seems like the wrong semantic since you're updating the media, not deleting it.

Reply:
@dklimpel so I did a straw poll among the team in #synapse-dev and the general feeling was that POST would be more intuitive than DELETE.

Although `DELETE` is in some ways the opposite of `POST`, I don't think that's always the case, and it's certainly not the case that there are any standards around that.

Reply:
Ok. Now it is a `POST` request.